config/strategies/mockStrategy.js

```diff
@@ -0,0 +1,62 @@
+import passport from 'passport';
+import util from 'util';
+
+let strategyCallback;
+let redirectURL;
+
+const mockUser = {
+  email: 'mockuser@gmail.com',
+  firstName: 'Johnmock',
+  lastName: 'Larry',
+  imageUrl: 'https://mock.com/mock/image/url'
+};
+
+/**
+ * @return {undefined}
+ * @param {*} options
+ * @param {*} callback
+ */
+function MockStrategy(options, callback) {
```

Comment:
This looks really good. 

However, I think we could do with a class here in place of this `MockStrategy` constructor function. That way, `authenticate` is just a class method, and `MockStrategy simply `extends passport.Strategy`, no need for node's `util` function.

Reply:
This is really better. The test coverage even increased.